finalizers/commandlogging.js

```diff
@@ -1,14 +1,13 @@
 const chalk = require("chalk");
 const now = require("performance-now");
 
-const clk = new chalk.constructor({ enabled: true });
```

Comment:
This has caused issues for me in your last pr, please revert this and the other instance you removed this from.